ibm/service/database/data_source_ibm_database_test.go

```diff
@@ -20,13 +21,17 @@ func TestAccIBMDatabaseDataSource_basic(t *testing.T) {
 	testName := fmt.Sprintf("tf-Pgress-%s", acctest.RandString(16))
 	dataName := "data.ibm_database." + testName
 	resourceName := "ibm_database.db"
+	location := os.Getenv("ICD_DB_REGION")
```

Comment:
Instead of adding the env in each place can we add that variable to acctest.go file and refer in every testcase
https://github.com/IBM-Cloud/terraform-provider-ibm/blob/master/ibm/acctest/acctest.go

Reply:
@hkantare thanks for the review! I've gone ahead and made the neccessary changes.